freedomofpress/securedrop-workstation
# Updates Whonix-based templates 14 -> 15

### Status

Ready for review.

Closes #300. Closes #306.

### Description

Updates all Whonix-related VMs to use Whonix 15 as base, rather than the EOL Whonix 14.  Leverages the new upgrade-in-place logic introduced in #352.

### Test plan
The upgrade-in-place scenario is the most important to test, since we want to ensure that automatic upgrades will handle the transition smoothly. 

- checkout latest master (7c19c6338900d21871471fce673d4e4479e0e07d at time of writing)
- `make clone && make clean && make all`
- checkout this PR branch
- `make clone && make all`
- `make test`

Confirm no errors.
